packages/@webex/contact-center/src/services/core/Utils.ts

```diff
@@ -130,3 +131,44 @@ export const createErrDetailsObject = (errObj: WebexRequestPayload) => {
 
   return new Err.Details('Service.reqs.generic.failure', details);
 };
+
+// Task destination utilities (shared across services)
+export const isEntryPointOrEpdn = (destAgentType?: string): boolean => {
+  return destAgentType === 'EPDN' || destAgentType === 'ENTRYPOINT';
+};
+
+export const getAgentActionTypeFromTask = (taskData?: TaskData): 'DIAL_NUMBER' | '' => {
+  const destAgentType = taskData?.destinationType;
+
+  return destAgentType === 'DN' || isEntryPointOrEpdn(destAgentType) ? 'DIAL_NUMBER' : '';
+};
+
+export const getDestAgentTypeForEporEpdn = (): ConsultTransferPayLoad['destinationType'] => {
+  return CONSULT_TRANSFER_DESTINATION_TYPE.ENTRYPOINT;
+};
+
+/**
+ * Derives the consult transfer destination type based on the provided task data.
+ *
+ * Logic parity with desktop behavior:
+ * - If agent action is dialing a number (DN/EPDN/ENTRYPOINT):
+ *   - ENTRYPOINT/EPDN map to ENTRYPOINT
+ *   - DN maps to DIALNUMBER
+ * - Otherwise defaults to AGENT
+ *
+ * @param taskData - The task data used to infer the agent action and destination type
+ * @returns The normalized destination type to be used for consult transfer
+ */
+export const deriveConsultTransferDestinationType = (
+  taskData?: TaskData
+): ConsultTransferPayLoad['destinationType'] => {
+  const agentActionType = getAgentActionTypeFromTask(taskData);
+
+  if (agentActionType === 'DIAL_NUMBER') {
+    return isEntryPointOrEpdn(taskData?.destinationType)
+      ? getDestAgentTypeForEporEpdn()
```

Comment:
We dont need a getter for this constant.
```suggestion
      ? CONSULT_TRANSFER_DESTINATION_TYPE.ENTRYPOINT
```